src/api/db/models/Image.js

```diff
@@ -100,6 +100,53 @@ export class ImageModel {
     }
   }
 
+  static async deleteImages(input, context) {
+    try {
+      const res = await Promise.allSettled(input.imageIds.map((imageId) => {
+        return this.deleteImage({ imageId }, context);
+      }));
+
+      const errors = res
+        .filter((r) => { return r.status === 'rejected'; })
+        .map((r) => { return r.reason; }); // Will always be an ApolloError
```

Comment:
What does that comment `// Will always be an ApolloError` mean? Is that because if anything goes wrong in the `deleteImage` try block, it will be caught and an `ApolloError` will be thrown?

Reply:
@nathanielrindlaub Per slack, the `r.reason` itself is always an instance of an ApolloError. I didn't see a straightforward way in the docs to throw multiple errors simultaneously so for the moment I've stayed with the current errors array output.